src/cmd/linuxkit/moby/docker.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/docker/cli/cli/config/configfile"
 	distref "github.com/docker/distribution/reference"
 	"github.com/docker/docker/api/types"
+	apitypes "github.com/docker/docker/api/types"
```

Comment:
you are including this twice with different names!

Reply:
elsewhere you include it as dockertypes...

Reply:
OK, standardized on `dockertypes`